utility/utils.py

```diff
@@ -756,24 +759,29 @@ def get_openshift_installer(version=defaults.INSTALLER_VERSION):
         # return to the previous working directory
         os.chdir(previous_dir)
 
-    add_path_to_env_path(defaults.BIN_DIR)
+    add_path_to_env_path(bin_dir)
 
     return installer_binary_path
 
 
-def get_openshift_client(version=defaults.CLIENT_VERSION):
+def get_openshift_client(
+    version=RUN['client_version'],
+    bin_dir=RUN['bin_dir'],
```

Comment:
As mentioned above, I would put both `client_version` and `bin_dir` variables also to the DEPLOYMENT section... but consider it just as my POV...

Reply:
Agreeing to previous similar comment, so resolving this.